docs/concept/#894-Efficient-orchestration-of-edc-negotiations/#894-Efficient-orchestration-of-edc-negotiations.md

```diff
@@ -0,0 +1,103 @@
+# #894 Efficient orchestration of edc negotiations
```

Comment:
In general there is a template: Please use this template in the future to have a common structure for concepts. https://github.com/eclipse-tractusx/item-relationship-service/blob/aeadd38e192525c89d563732e7266d09afc88bd7/docs/concept/TEMPLATE_Concept.md